src/Goteo/Console/Command/RoundCommand.php

```diff
@@ -148,22 +148,24 @@ protected function execute(InputInterface $input, OutputInterface $output) {
                     if ($skip_invests) {
                         $this->warning("Skipping Invests refunds as required\n--");
                     } else {
-                        // Execute "console refund"  command for returning invests
+                        // Execute "console refund" command for returning invests
                         // refund Invests for this project
                         $args = ' --project ' . escapeshellarg($project->id);
                         if($update) $args .= ' --update';
                         else        $args .= ' --any-project';
                         if($output->isVerbose()) $args .= ' --verbose';
-                        $process = new Process(GOTEO_PATH . 'bin/console refund' . $args);
-                        $process->setTimeout(null); // no time limit
+
+                        $process = new Process([GOTEO_PATH . 'bin/console refund' . $args]);
```

Comment:
SF 4.2 => `ProcessHelper` has changed how to run commands. It requires passing the command and arguments in an array. Check if it continues working.